Ministry.java

```diff
@@ -51,6 +51,8 @@ public Ministry(String name, int level)
 	* Checks the approval status of a venue
 	* @param a 
```

Comment:
When you say @param a , say what the parameter is about. For e.g. @param a - This is an approval request

Reply:
Okay I'm done then